nbs/directives.ipynb

```diff
@@ -0,0 +1,828 @@
+{
```

Comment:
<p>"When a cell does not have this annotation, the output is fixed ...." </p><p> </p><p>should be</p><p> </p><p>"When a cell does not have this annotation, it is run according to the default rules [described here](https://nbdev.fast.ai/explanations/showdoc.html)."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>